syz-manager: panic: runtime error: invalid memory address or nil pointer dereference

After updating to 9589d1d980cc ("mod: bump k8s.io/apimachinery from 0.36.2 to 0.36.3"), `syz-manager` crashes with the following report:
```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0x8df559]
```
Compiled with Go 1.26.4. Full `syz-manager` output attached.

[log.txt](https://github.com/user-attachments/files/30668869/log.txt)

--- a/pkg/corpus/prio.go
+++ b/pkg/corpus/prio.go
@@ -79,8 +79,7 @@ func (corpus *Corpus) Programs() []*prog.Prog {
 }
 
 func (pl *ProgramsList) clear() {
-	clear(pl.progs)
-	pl.progs = pl.progs[:0]
-	pl.accPrios = pl.accPrios[:0]
+	pl.progs = nil
+	pl.accPrios = nil
 	pl.sumPrios = 0
 }

